scripts/version/README.md

```diff
@@ -0,0 +1,34 @@
+## How sonar-dotnet version is set in our pipeline
+
+During the pipeline run, we promote the following artifacts:
+
+1. `sonar-csharp-plugin-x.y.z.buildNumber.jar`
+1. `sonar-vbnet-plugin-x.y.z.buildNumber.jar`
+1. `SonarAnalyzer.CSharp-x.y.z.nupkg`
+1. `SonarAnalyzer.VisualBasic-x.y.z.nupkg`
+1. `SonarAnalyzer.CFG-x.y.z.nupkg` (promoted only on repox)
+
+### How is the version set for the *.jar artifacts
+
+The Java artifacts' major, minor and, patch parts of the version, are set in the [sonar-dotnet pom.xml](https://github.com/SonarSource/sonar-dotnet/blob/master/pom.xml#L14).
+The `buildNumber` is set in the Azure pipeline [here](https://github.com/SonarSource/sonar-dotnet/blob/master/azure-pipelines.yml#L538) with the use of the `update-maven-version-steps.yml` and it's equal to the pipeline [`buildId`](https://learn.microsoft.com/en-us/azure/devops/pipelines/process/run-number?view=azure-devops&tabs=yaml#tokens).
```

Comment:
The `buildId` information doesn't seem to be correct. As buildId is only the `buildNumber`, while the sentence speaks about the whole version.

What is the relation between full version and buildId should probably be explain in a leading chapter, before Java/.NET sections..